\documentclass[10pt,twoside,english]{scrartcl}

\usepackage[T1]{fontenc}
\usepackage[latin9]{inputenc}
\usepackage[letterpaper]{geometry}
\geometry{verbose,tmargin=2cm,bmargin=2cm,lmargin=2cm,rmargin=2cm,footskip=1cm}
\usepackage{fancyhdr}
\pagestyle{fancy}
\setlength{\parindent}{0cm}
\usepackage{babel}
\usepackage[unicode=true,
 bookmarks=true,bookmarksnumbered=false,bookmarksopen=false,
 breaklinks=false,pdfborder={0 0 0},backref=false,colorlinks=true]
 {hyperref}
\hypersetup{pdftitle={Test Report},
 pdfauthor={Louis-Philippe Lemieux Perreault},
 linkcolor={link_blue},citecolor={link_blue},urlcolor={link_blue}}

\makeatletter
\usepackage[activate={true,nocompatibility},final,tracking=true,kerning=true,spacing=true,factor=1100,stretch=10,shrink=10]{microtype}

% Some color definitions
\usepackage{xcolor}
\definecolor{blue}{HTML}{0099CC}
\definecolor{red}{HTML}{CC0000}
\definecolor{green}{HTML}{669900}
\definecolor{link_blue}{HTML}{21759B}
\definecolor{light_gray}{HTML}{BABABA}

% Floats, caption and subcaptions
\usepackage{float}
\usepackage{graphicx}
\usepackage[bf]{caption}

% Ordinal numbers
\usepackage{engord}

% To use the path command
\usepackage{url}

% The Report number...
\newcommand{\ReportNumber}{Test Report}

% Fancy header
\pagestyle{fancy}
\fancyhf{}
\fancyfoot[RO,LE]{\footnotesize{\thepage}}
\fancyfoot[LO,RE]{\footnotesize{\ReportNumber}}
\fancyfoot[C]{\footnotesize{Automatically generated by \texttt{genipe} version 1.4.2}}
\renewcommand{\headrulewidth}{0pt}
\renewcommand{\footrulewidth}{1pt}

% The plain fancy header
\fancypagestyle{plain}{
    \fancyhf{} % clear all header and footer fields
    \fancyfoot[RO,LE]{\footnotesize{\thepage}}
    \fancyfoot[LO,RE]{\footnotesize{\ReportNumber}}
    \fancyfoot[C]{\footnotesize{Automatically generated by \texttt{genipe} version 1.4.2}}
    \renewcommand{\headrulewidth}{0pt}
    \renewcommand{\footrulewidth}{1pt}
}

% Table's numerotations are in Roman numbers
\renewcommand{\thetable}{\Roman{table}}

\let\tempone\itemize
\let\temptwo\enditemize
\renewenvironment{itemize}{\tempone\setlength{\itemsep}{0pt}}{\temptwo}

\let\ttempone\enumerate
\let\ttemptwo\endenumerate
\renewenvironment{enumerate}{\ttempone\setlength{\itemsep}{0pt}}{\ttemptwo}

\let\tttempone\description
\let\tttemptwo\enddescription
\renewenvironment{description}{\tttempone\setlength{\itemsep}{0pt}}{\tttemptwo}

\makeatother

\begin{document}

\title{Tutorial}


\subtitle{\ReportNumber}


\author{Automatically generated by genipe}


\date{August \engordnumber{25}, 2019}

\maketitle
\microtypesetup{protrusion=false} % disables protrusion locally in the document

\tableofcontents{}

\microtypesetup{protrusion=true} % enables protrusion



\section{Background}\label{sec:background}

The aim of this project is to perform genome-wide imputation using the study cohort.

\section{Methods}\label{sec:methods}


The following (cleaned) files provided information about the study cohort
dataset for 1,465 samples and 366,251
markers (including 0 markers located on sexual or
mitochondrial chromosomes):

\begin{itemize}
  \item \path{/home/zhangchi/f/genipe_tutorial/data/merge_uniq/sample1_uniq.bed}
  \item \path{/home/zhangchi/f/genipe_tutorial/data/merge_uniq/sample1_uniq.bim}
  \item \path{/home/zhangchi/f/genipe_tutorial/data/merge_uniq/sample1_uniq.fam}
\end{itemize}

IMPUTE2's pre-phasing approach can work with phased haplotypes from SHAPEIT, a
highly accurate phasing algorithm that can handle mixtures of unrelated
samples, duos or trios. The usage of SHAPEIT is highly recommended to infer
haplotypes underlying the study genotypes. The phased haplotypes are then
passed to IMPUTE2 for imputation. Although pre-phasing allows for very fast
imputation, it leads to a small loss in accuracy since the estimation
uncertainty in the study haplotypes is ignored. SHAPEIT version
v2.r904~\cite{Delaneau13_23269371} and IMPUTE2 version
2.3.2~\cite{Howie09_19543373,Howie11_22384356,Howie12_22820512}
were used for this analysis. Binary pedfiles were processed using Plink version
unknown~\cite{Purcell07_17701901}.\\

To speed up the pre-phasing and imputation steps, the dataset was split by
chromosome. The following quality steps were then performed on each chromosome:


\begin{enumerate}

\item Ambiguous markers with alleles \texttt{A}/\texttt{T} and
\texttt{C}/\texttt{G}, duplicated markers (same position), and markers
located on the mitochondrial or the Y chromosomes were excluded from
the imputation. \textbf{In total, 1,526 ambiguous, 37 duplicated and 0
Y/mitochondrial markers were excluded.}


\item Markers' strand was checked using the SHAPEIT algorithm and IMPUTE2's
reference files. \textbf{In total, 34,751 markers had an incorrect
strand and were flipped using Plink.}


\item The strand of each marker was checked again using SHAPEIT against
IMPUTE2's reference files. \textbf{In total, 38 markers were found to
still be on the wrong strand, and were hence excluded from the final
dataset using Plink.}

\end{enumerate}



\textbf{In total, 364,650 markers were used for phasing
using SHAPEIT.} IMPUTE2 was then used to impute markers genome-wide using its
reference file (filtering out sites where \texttt{ALL<0.01} or \texttt{ALL>0.99}).

\section{Results}\label{sec:results}


\subsection{Cross-validation}\label{subsec:cross_validation}


According to IMPUTE2's documentation, the cross-validation tables are "based on
an internal cross-validation that is performed during each IMPUTE2 run. For
this analysis, the program masks the genotypes of one variant at a time in the
study data and imputes the masked genotypes by using the remaining study and
reference data. The imputed genotypes are then compared with the original
genotypes to produce the concordance statistics."\\

Tables~\ref{tab:cross_validation_chr_1} to
\ref{tab:cross_validation_chr_22} show the cross-validation
results for the analyzed chromosomes. Table~\ref{tab:cross_validation} shows
the cross-validation results across the autosomes.


\begin{table}[H]
\protect\caption{IMPUTE2's internal cross-validation for chromosome 1. Tables show the
percentage of concordance between genotyped calls and imputed calls
for 38,497,270 genotypes.\label{tab:cross_validation_chr_1}}

\centering

\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Nb Geno}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[0.0-0.1]$ & 0 & 0.0\\
$[0.1-0.2]$ & 0 & 0.0\\
$[0.2-0.3]$ & 0 & 0.0\\
$[0.3-0.4]$ & 0 & 0.0\\
$[0.4-0.5]$ & 84,798 & 44.5\\
$[0.5-0.6]$ & 724,450 & 53.1\\
$[0.6-0.7]$ & 728,289 & 61.0\\
$[0.7-0.8]$ & 869,782 & 69.4\\
$[0.8-0.9]$ & 1,325,112 & 78.2\\
$[0.9-1.0]$ & 34,764,839 & 97.5\\
\hline 
\end{tabular}
\hfill
\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Called (\%)}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[\geq 0.0]$ & 100.0 & 94.5\\
$[\geq 0.1]$ & 100.0 & 94.5\\
$[\geq 0.2]$ & 100.0 & 94.5\\
$[\geq 0.3]$ & 100.0 & 94.5\\
$[\geq 0.4]$ & 100.0 & 94.5\\
$[\geq 0.5]$ & 99.8 & 94.7\\
$[\geq 0.6]$ & 97.9 & 95.5\\
$[\geq 0.7]$ & 96.0 & 96.1\\
$[\geq 0.8]$ & 93.7 & 96.8\\
$[\geq 0.9]$ & 90.3 & 97.5\\
\hline 
\end{tabular}


\end{table}

\begin{table}[H]
\protect\caption{IMPUTE2's internal cross-validation for chromosome 2. Tables show the
percentage of concordance between genotyped calls and imputed calls
for 39,801,120 genotypes.\label{tab:cross_validation_chr_2}}

\centering

\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Nb Geno}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[0.0-0.1]$ & 0 & 0.0\\
$[0.1-0.2]$ & 0 & 0.0\\
$[0.2-0.3]$ & 0 & 0.0\\
$[0.3-0.4]$ & 0 & 0.0\\
$[0.4-0.5]$ & 75,582 & 44.4\\
$[0.5-0.6]$ & 681,272 & 53.2\\
$[0.6-0.7]$ & 685,606 & 61.0\\
$[0.7-0.8]$ & 824,112 & 69.7\\
$[0.8-0.9]$ & 1,264,453 & 78.5\\
$[0.9-1.0]$ & 36,270,095 & 97.7\\
\hline 
\end{tabular}
\hfill
\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Called (\%)}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[\geq 0.0]$ & 100.0 & 95.0\\
$[\geq 0.1]$ & 100.0 & 95.0\\
$[\geq 0.2]$ & 100.0 & 95.0\\
$[\geq 0.3]$ & 100.0 & 95.0\\
$[\geq 0.4]$ & 100.0 & 95.0\\
$[\geq 0.5]$ & 99.8 & 95.1\\
$[\geq 0.6]$ & 98.1 & 95.8\\
$[\geq 0.7]$ & 96.4 & 96.4\\
$[\geq 0.8]$ & 94.3 & 97.0\\
$[\geq 0.9]$ & 91.1 & 97.7\\
\hline 
\end{tabular}


\end{table}

\begin{table}[H]
\protect\caption{IMPUTE2's internal cross-validation for chromosome 3. Tables show the
percentage of concordance between genotyped calls and imputed calls
for 34,468,520 genotypes.\label{tab:cross_validation_chr_3}}

\centering

\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Nb Geno}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[0.0-0.1]$ & 0 & 0.0\\
$[0.1-0.2]$ & 0 & 0.0\\
$[0.2-0.3]$ & 0 & 0.0\\
$[0.3-0.4]$ & 0 & 0.0\\
$[0.4-0.5]$ & 67,604 & 45.0\\
$[0.5-0.6]$ & 597,175 & 53.4\\
$[0.6-0.7]$ & 603,941 & 61.2\\
$[0.7-0.8]$ & 722,790 & 69.6\\
$[0.8-0.9]$ & 1,107,580 & 78.6\\
$[0.9-1.0]$ & 31,369,430 & 97.7\\
\hline 
\end{tabular}
\hfill
\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Called (\%)}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[\geq 0.0]$ & 100.0 & 95.0\\
$[\geq 0.1]$ & 100.0 & 95.0\\
$[\geq 0.2]$ & 100.0 & 95.0\\
$[\geq 0.3]$ & 100.0 & 95.0\\
$[\geq 0.4]$ & 100.0 & 95.0\\
$[\geq 0.5]$ & 99.8 & 95.1\\
$[\geq 0.6]$ & 98.1 & 95.8\\
$[\geq 0.7]$ & 96.3 & 96.4\\
$[\geq 0.8]$ & 94.2 & 97.0\\
$[\geq 0.9]$ & 91.0 & 97.7\\
\hline 
\end{tabular}


\end{table}

\begin{table}[H]
\protect\caption{IMPUTE2's internal cross-validation for chromosome 4. Tables show the
percentage of concordance between genotyped calls and imputed calls
for 31,605,910 genotypes.\label{tab:cross_validation_chr_4}}

\centering

\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Nb Geno}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[0.0-0.1]$ & 0 & 0.0\\
$[0.1-0.2]$ & 0 & 0.0\\
$[0.2-0.3]$ & 0 & 0.0\\
$[0.3-0.4]$ & 0 & 0.0\\
$[0.4-0.5]$ & 64,503 & 44.5\\
$[0.5-0.6]$ & 562,727 & 53.0\\
$[0.6-0.7]$ & 566,067 & 61.0\\
$[0.7-0.8]$ & 680,254 & 69.5\\
$[0.8-0.9]$ & 1,035,654 & 78.4\\
$[0.9-1.0]$ & 28,696,705 & 97.6\\
\hline 
\end{tabular}
\hfill
\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Called (\%)}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[\geq 0.0]$ & 100.0 & 94.8\\
$[\geq 0.1]$ & 100.0 & 94.8\\
$[\geq 0.2]$ & 100.0 & 94.8\\
$[\geq 0.3]$ & 100.0 & 94.8\\
$[\geq 0.4]$ & 100.0 & 94.8\\
$[\geq 0.5]$ & 99.8 & 94.9\\
$[\geq 0.6]$ & 98.0 & 95.6\\
$[\geq 0.7]$ & 96.2 & 96.3\\
$[\geq 0.8]$ & 94.1 & 96.9\\
$[\geq 0.9]$ & 90.8 & 97.6\\
\hline 
\end{tabular}


\end{table}

\begin{table}[H]
\protect\caption{IMPUTE2's internal cross-validation for chromosome 5. Tables show the
percentage of concordance between genotyped calls and imputed calls
for 28,823,875 genotypes.\label{tab:cross_validation_chr_5}}

\centering

\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Nb Geno}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[0.0-0.1]$ & 0 & 0.0\\
$[0.1-0.2]$ & 0 & 0.0\\
$[0.2-0.3]$ & 0 & 0.0\\
$[0.3-0.4]$ & 0 & 0.0\\
$[0.4-0.5]$ & 62,264 & 44.3\\
$[0.5-0.6]$ & 539,553 & 52.9\\
$[0.6-0.7]$ & 540,695 & 61.0\\
$[0.7-0.8]$ & 650,031 & 69.2\\
$[0.8-0.9]$ & 989,738 & 78.3\\
$[0.9-1.0]$ & 26,041,594 & 97.5\\
\hline 
\end{tabular}
\hfill
\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Called (\%)}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[\geq 0.0]$ & 100.0 & 94.5\\
$[\geq 0.1]$ & 100.0 & 94.5\\
$[\geq 0.2]$ & 100.0 & 94.5\\
$[\geq 0.3]$ & 100.0 & 94.5\\
$[\geq 0.4]$ & 100.0 & 94.5\\
$[\geq 0.5]$ & 99.8 & 94.6\\
$[\geq 0.6]$ & 97.9 & 95.4\\
$[\geq 0.7]$ & 96.0 & 96.1\\
$[\geq 0.8]$ & 93.8 & 96.8\\
$[\geq 0.9]$ & 90.4 & 97.5\\
\hline 
\end{tabular}


\end{table}

\begin{table}[H]
\protect\caption{IMPUTE2's internal cross-validation for chromosome 6. Tables show the
percentage of concordance between genotyped calls and imputed calls
for 37,694,450 genotypes.\label{tab:cross_validation_chr_6}}

\centering

\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Nb Geno}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[0.0-0.1]$ & 0 & 0.0\\
$[0.1-0.2]$ & 0 & 0.0\\
$[0.2-0.3]$ & 0 & 0.0\\
$[0.3-0.4]$ & 0 & 0.0\\
$[0.4-0.5]$ & 53,580 & 44.3\\
$[0.5-0.6]$ & 488,549 & 53.2\\
$[0.6-0.7]$ & 495,464 & 61.3\\
$[0.7-0.8]$ & 600,069 & 70.1\\
$[0.8-0.9]$ & 922,318 & 79.0\\
$[0.9-1.0]$ & 35,134,470 & 98.2\\
\hline 
\end{tabular}
\hfill
\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Called (\%)}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[\geq 0.0]$ & 100.0 & 96.2\\
$[\geq 0.1]$ & 100.0 & 96.2\\
$[\geq 0.2]$ & 100.0 & 96.2\\
$[\geq 0.3]$ & 100.0 & 96.2\\
$[\geq 0.4]$ & 100.0 & 96.2\\
$[\geq 0.5]$ & 99.9 & 96.3\\
$[\geq 0.6]$ & 98.6 & 96.8\\
$[\geq 0.7]$ & 97.2 & 97.3\\
$[\geq 0.8]$ & 95.7 & 97.7\\
$[\geq 0.9]$ & 93.2 & 98.2\\
\hline 
\end{tabular}


\end{table}

\begin{table}[H]
\protect\caption{IMPUTE2's internal cross-validation for chromosome 7. Tables show the
percentage of concordance between genotyped calls and imputed calls
for 27,799,840 genotypes.\label{tab:cross_validation_chr_7}}

\centering

\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Nb Geno}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[0.0-0.1]$ & 0 & 0.0\\
$[0.1-0.2]$ & 0 & 0.0\\
$[0.2-0.3]$ & 0 & 0.0\\
$[0.3-0.4]$ & 0 & 0.0\\
$[0.4-0.5]$ & 54,734 & 44.5\\
$[0.5-0.6]$ & 498,387 & 53.3\\
$[0.6-0.7]$ & 503,927 & 61.4\\
$[0.7-0.8]$ & 603,831 & 69.9\\
$[0.8-0.9]$ & 929,155 & 78.8\\
$[0.9-1.0]$ & 25,209,806 & 97.6\\
\hline 
\end{tabular}
\hfill
\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Called (\%)}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[\geq 0.0]$ & 100.0 & 94.8\\
$[\geq 0.1]$ & 100.0 & 94.8\\
$[\geq 0.2]$ & 100.0 & 94.8\\
$[\geq 0.3]$ & 100.0 & 94.8\\
$[\geq 0.4]$ & 100.0 & 94.8\\
$[\geq 0.5]$ & 99.8 & 94.9\\
$[\geq 0.6]$ & 98.0 & 95.7\\
$[\geq 0.7]$ & 96.2 & 96.3\\
$[\geq 0.8]$ & 94.0 & 96.9\\
$[\geq 0.9]$ & 90.7 & 97.6\\
\hline 
\end{tabular}


\end{table}

\begin{table}[H]
\protect\caption{IMPUTE2's internal cross-validation for chromosome 8. Tables show the
percentage of concordance between genotyped calls and imputed calls
for 25,587,690 genotypes.\label{tab:cross_validation_chr_8}}

\centering

\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Nb Geno}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[0.0-0.1]$ & 0 & 0.0\\
$[0.1-0.2]$ & 0 & 0.0\\
$[0.2-0.3]$ & 0 & 0.0\\
$[0.3-0.4]$ & 0 & 0.0\\
$[0.4-0.5]$ & 50,554 & 44.6\\
$[0.5-0.6]$ & 455,186 & 53.3\\
$[0.6-0.7]$ & 457,606 & 61.2\\
$[0.7-0.8]$ & 550,538 & 69.8\\
$[0.8-0.9]$ & 840,282 & 78.7\\
$[0.9-1.0]$ & 23,233,524 & 97.6\\
\hline 
\end{tabular}
\hfill
\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Called (\%)}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[\geq 0.0]$ & 100.0 & 94.8\\
$[\geq 0.1]$ & 100.0 & 94.8\\
$[\geq 0.2]$ & 100.0 & 94.8\\
$[\geq 0.3]$ & 100.0 & 94.8\\
$[\geq 0.4]$ & 100.0 & 94.8\\
$[\geq 0.5]$ & 99.8 & 94.9\\
$[\geq 0.6]$ & 98.0 & 95.7\\
$[\geq 0.7]$ & 96.2 & 96.3\\
$[\geq 0.8]$ & 94.1 & 96.9\\
$[\geq 0.9]$ & 90.8 & 97.6\\
\hline 
\end{tabular}


\end{table}

\begin{table}[H]
\protect\caption{IMPUTE2's internal cross-validation for chromosome 9. Tables show the
percentage of concordance between genotyped calls and imputed calls
for 21,701,045 genotypes.\label{tab:cross_validation_chr_9}}

\centering

\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Nb Geno}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[0.0-0.1]$ & 0 & 0.0\\
$[0.1-0.2]$ & 0 & 0.0\\
$[0.2-0.3]$ & 0 & 0.0\\
$[0.3-0.4]$ & 0 & 0.0\\
$[0.4-0.5]$ & 51,048 & 45.1\\
$[0.5-0.6]$ & 432,926 & 53.1\\
$[0.6-0.7]$ & 432,673 & 61.0\\
$[0.7-0.8]$ & 519,103 & 69.2\\
$[0.8-0.9]$ & 795,866 & 77.9\\
$[0.9-1.0]$ & 19,469,429 & 97.2\\
\hline 
\end{tabular}
\hfill
\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Called (\%)}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[\geq 0.0]$ & 100.0 & 94.1\\
$[\geq 0.1]$ & 100.0 & 94.1\\
$[\geq 0.2]$ & 100.0 & 94.1\\
$[\geq 0.3]$ & 100.0 & 94.1\\
$[\geq 0.4]$ & 100.0 & 94.1\\
$[\geq 0.5]$ & 99.8 & 94.2\\
$[\geq 0.6]$ & 97.8 & 95.1\\
$[\geq 0.7]$ & 95.8 & 95.8\\
$[\geq 0.8]$ & 93.4 & 96.5\\
$[\geq 0.9]$ & 89.7 & 97.2\\
\hline 
\end{tabular}


\end{table}

\begin{table}[H]
\protect\caption{IMPUTE2's internal cross-validation for chromosome 10. Tables show the
percentage of concordance between genotyped calls and imputed calls
for 25,110,100 genotypes.\label{tab:cross_validation_chr_10}}

\centering

\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Nb Geno}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[0.0-0.1]$ & 0 & 0.0\\
$[0.1-0.2]$ & 0 & 0.0\\
$[0.2-0.3]$ & 0 & 0.0\\
$[0.3-0.4]$ & 0 & 0.0\\
$[0.4-0.5]$ & 51,764 & 44.2\\
$[0.5-0.6]$ & 456,959 & 53.1\\
$[0.6-0.7]$ & 458,561 & 61.2\\
$[0.7-0.8]$ & 549,053 & 69.4\\
$[0.8-0.9]$ & 842,717 & 78.5\\
$[0.9-1.0]$ & 22,751,046 & 97.5\\
\hline 
\end{tabular}
\hfill
\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Called (\%)}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[\geq 0.0]$ & 100.0 & 94.7\\
$[\geq 0.1]$ & 100.0 & 94.7\\
$[\geq 0.2]$ & 100.0 & 94.7\\
$[\geq 0.3]$ & 100.0 & 94.7\\
$[\geq 0.4]$ & 100.0 & 94.7\\
$[\geq 0.5]$ & 99.8 & 94.8\\
$[\geq 0.6]$ & 98.0 & 95.6\\
$[\geq 0.7]$ & 96.1 & 96.2\\
$[\geq 0.8]$ & 94.0 & 96.9\\
$[\geq 0.9]$ & 90.6 & 97.5\\
\hline 
\end{tabular}


\end{table}

\begin{table}[H]
\protect\caption{IMPUTE2's internal cross-validation for chromosome 11. Tables show the
percentage of concordance between genotyped calls and imputed calls
for 24,040,650 genotypes.\label{tab:cross_validation_chr_11}}

\centering

\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Nb Geno}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[0.0-0.1]$ & 0 & 0.0\\
$[0.1-0.2]$ & 0 & 0.0\\
$[0.2-0.3]$ & 0 & 0.0\\
$[0.3-0.4]$ & 0 & 0.0\\
$[0.4-0.5]$ & 47,269 & 43.9\\
$[0.5-0.6]$ & 419,539 & 53.2\\
$[0.6-0.7]$ & 422,796 & 61.3\\
$[0.7-0.8]$ & 506,009 & 69.5\\
$[0.8-0.9]$ & 781,034 & 78.6\\
$[0.9-1.0]$ & 21,864,003 & 97.6\\
\hline 
\end{tabular}
\hfill
\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Called (\%)}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[\geq 0.0]$ & 100.0 & 94.9\\
$[\geq 0.1]$ & 100.0 & 94.9\\
$[\geq 0.2]$ & 100.0 & 94.9\\
$[\geq 0.3]$ & 100.0 & 94.9\\
$[\geq 0.4]$ & 100.0 & 94.9\\
$[\geq 0.5]$ & 99.8 & 95.0\\
$[\geq 0.6]$ & 98.1 & 95.7\\
$[\geq 0.7]$ & 96.3 & 96.3\\
$[\geq 0.8]$ & 94.2 & 97.0\\
$[\geq 0.9]$ & 91.0 & 97.6\\
\hline 
\end{tabular}


\end{table}

\begin{table}[H]
\protect\caption{IMPUTE2's internal cross-validation for chromosome 12. Tables show the
percentage of concordance between genotyped calls and imputed calls
for 24,068,485 genotypes.\label{tab:cross_validation_chr_12}}

\centering

\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Nb Geno}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[0.0-0.1]$ & 0 & 0.0\\
$[0.1-0.2]$ & 0 & 0.0\\
$[0.2-0.3]$ & 0 & 0.0\\
$[0.3-0.4]$ & 0 & 0.0\\
$[0.4-0.5]$ & 51,459 & 44.6\\
$[0.5-0.6]$ & 450,807 & 53.0\\
$[0.6-0.7]$ & 450,707 & 61.0\\
$[0.7-0.8]$ & 541,749 & 69.4\\
$[0.8-0.9]$ & 825,125 & 78.4\\
$[0.9-1.0]$ & 21,748,638 & 97.5\\
\hline 
\end{tabular}
\hfill
\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Called (\%)}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[\geq 0.0]$ & 100.0 & 94.6\\
$[\geq 0.1]$ & 100.0 & 94.6\\
$[\geq 0.2]$ & 100.0 & 94.6\\
$[\geq 0.3]$ & 100.0 & 94.6\\
$[\geq 0.4]$ & 100.0 & 94.6\\
$[\geq 0.5]$ & 99.8 & 94.7\\
$[\geq 0.6]$ & 97.9 & 95.5\\
$[\geq 0.7]$ & 96.0 & 96.2\\
$[\geq 0.8]$ & 93.8 & 96.8\\
$[\geq 0.9]$ & 90.4 & 97.5\\
\hline 
\end{tabular}


\end{table}

\begin{table}[H]
\protect\caption{IMPUTE2's internal cross-validation for chromosome 13. Tables show the
percentage of concordance between genotyped calls and imputed calls
for 18,207,020 genotypes.\label{tab:cross_validation_chr_13}}

\centering

\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Nb Geno}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[0.0-0.1]$ & 0 & 0.0\\
$[0.1-0.2]$ & 0 & 0.0\\
$[0.2-0.3]$ & 0 & 0.0\\
$[0.3-0.4]$ & 0 & 0.0\\
$[0.4-0.5]$ & 38,588 & 43.9\\
$[0.5-0.6]$ & 328,722 & 53.0\\
$[0.6-0.7]$ & 331,442 & 61.0\\
$[0.7-0.8]$ & 399,418 & 69.5\\
$[0.8-0.9]$ & 609,157 & 78.4\\
$[0.9-1.0]$ & 16,499,693 & 97.6\\
\hline 
\end{tabular}
\hfill
\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Called (\%)}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[\geq 0.0]$ & 100.0 & 94.7\\
$[\geq 0.1]$ & 100.0 & 94.7\\
$[\geq 0.2]$ & 100.0 & 94.7\\
$[\geq 0.3]$ & 100.0 & 94.7\\
$[\geq 0.4]$ & 100.0 & 94.7\\
$[\geq 0.5]$ & 99.8 & 94.8\\
$[\geq 0.6]$ & 98.0 & 95.6\\
$[\geq 0.7]$ & 96.2 & 96.3\\
$[\geq 0.8]$ & 94.0 & 96.9\\
$[\geq 0.9]$ & 90.6 & 97.6\\
\hline 
\end{tabular}


\end{table}

\begin{table}[H]
\protect\caption{IMPUTE2's internal cross-validation for chromosome 14. Tables show the
percentage of concordance between genotyped calls and imputed calls
for 16,349,400 genotypes.\label{tab:cross_validation_chr_14}}

\centering

\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Nb Geno}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[0.0-0.1]$ & 0 & 0.0\\
$[0.1-0.2]$ & 0 & 0.0\\
$[0.2-0.3]$ & 0 & 0.0\\
$[0.3-0.4]$ & 0 & 0.0\\
$[0.4-0.5]$ & 37,700 & 43.8\\
$[0.5-0.6]$ & 314,060 & 52.7\\
$[0.6-0.7]$ & 316,120 & 60.7\\
$[0.7-0.8]$ & 377,468 & 69.2\\
$[0.8-0.9]$ & 575,139 & 78.1\\
$[0.9-1.0]$ & 14,728,913 & 97.4\\
\hline 
\end{tabular}
\hfill
\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Called (\%)}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[\geq 0.0]$ & 100.0 & 94.4\\
$[\geq 0.1]$ & 100.0 & 94.4\\
$[\geq 0.2]$ & 100.0 & 94.4\\
$[\geq 0.3]$ & 100.0 & 94.4\\
$[\geq 0.4]$ & 100.0 & 94.4\\
$[\geq 0.5]$ & 99.8 & 94.5\\
$[\geq 0.6]$ & 97.8 & 95.3\\
$[\geq 0.7]$ & 95.9 & 96.0\\
$[\geq 0.8]$ & 93.6 & 96.7\\
$[\geq 0.9]$ & 90.1 & 97.4\\
\hline 
\end{tabular}


\end{table}

\begin{table}[H]
\protect\caption{IMPUTE2's internal cross-validation for chromosome 15. Tables show the
percentage of concordance between genotyped calls and imputed calls
for 15,685,755 genotypes.\label{tab:cross_validation_chr_15}}

\centering

\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Nb Geno}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[0.0-0.1]$ & 0 & 0.0\\
$[0.1-0.2]$ & 0 & 0.0\\
$[0.2-0.3]$ & 0 & 0.0\\
$[0.3-0.4]$ & 0 & 0.0\\
$[0.4-0.5]$ & 42,682 & 44.7\\
$[0.5-0.6]$ & 341,611 & 52.6\\
$[0.6-0.7]$ & 339,771 & 60.3\\
$[0.7-0.8]$ & 402,498 & 68.4\\
$[0.8-0.9]$ & 606,854 & 77.1\\
$[0.9-1.0]$ & 13,952,339 & 97.0\\
\hline 
\end{tabular}
\hfill
\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Called (\%)}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[\geq 0.0]$ & 100.0 & 93.6\\
$[\geq 0.1]$ & 100.0 & 93.6\\
$[\geq 0.2]$ & 100.0 & 93.6\\
$[\geq 0.3]$ & 100.0 & 93.6\\
$[\geq 0.4]$ & 100.0 & 93.6\\
$[\geq 0.5]$ & 99.7 & 93.8\\
$[\geq 0.6]$ & 97.5 & 94.7\\
$[\geq 0.7]$ & 95.4 & 95.4\\
$[\geq 0.8]$ & 92.8 & 96.2\\
$[\geq 0.9]$ & 88.9 & 97.0\\
\hline 
\end{tabular}


\end{table}

\begin{table}[H]
\protect\caption{IMPUTE2's internal cross-validation for chromosome 16. Tables show the
percentage of concordance between genotyped calls and imputed calls
for 16,349,400 genotypes.\label{tab:cross_validation_chr_16}}

\centering

\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Nb Geno}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[0.0-0.1]$ & 0 & 0.0\\
$[0.1-0.2]$ & 0 & 0.0\\
$[0.2-0.3]$ & 0 & 0.0\\
$[0.3-0.4]$ & 0 & 0.0\\
$[0.4-0.5]$ & 44,139 & 44.9\\
$[0.5-0.6]$ & 362,445 & 52.9\\
$[0.6-0.7]$ & 361,654 & 60.7\\
$[0.7-0.8]$ & 430,057 & 68.7\\
$[0.8-0.9]$ & 648,990 & 77.4\\
$[0.9-1.0]$ & 14,502,115 & 96.9\\
\hline 
\end{tabular}
\hfill
\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Called (\%)}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[\geq 0.0]$ & 100.0 & 93.5\\
$[\geq 0.1]$ & 100.0 & 93.5\\
$[\geq 0.2]$ & 100.0 & 93.5\\
$[\geq 0.3]$ & 100.0 & 93.5\\
$[\geq 0.4]$ & 100.0 & 93.5\\
$[\geq 0.5]$ & 99.7 & 93.6\\
$[\geq 0.6]$ & 97.5 & 94.6\\
$[\geq 0.7]$ & 95.3 & 95.3\\
$[\geq 0.8]$ & 92.7 & 96.1\\
$[\geq 0.9]$ & 88.7 & 96.9\\
\hline 
\end{tabular}


\end{table}

\begin{table}[H]
\protect\caption{IMPUTE2's internal cross-validation for chromosome 17. Tables show the
percentage of concordance between genotyped calls and imputed calls
for 14,541,590 genotypes.\label{tab:cross_validation_chr_17}}

\centering

\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Nb Geno}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[0.0-0.1]$ & 0 & 0.0\\
$[0.1-0.2]$ & 0 & 0.0\\
$[0.2-0.3]$ & 0 & 0.0\\
$[0.3-0.4]$ & 0 & 0.0\\
$[0.4-0.5]$ & 47,357 & 44.7\\
$[0.5-0.6]$ & 359,603 & 52.5\\
$[0.6-0.7]$ & 354,868 & 60.1\\
$[0.7-0.8]$ & 420,101 & 68.0\\
$[0.8-0.9]$ & 626,058 & 76.5\\
$[0.9-1.0]$ & 12,733,603 & 96.7\\
\hline 
\end{tabular}
\hfill
\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Called (\%)}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[\geq 0.0]$ & 100.0 & 92.8\\
$[\geq 0.1]$ & 100.0 & 92.8\\
$[\geq 0.2]$ & 100.0 & 92.8\\
$[\geq 0.3]$ & 100.0 & 92.8\\
$[\geq 0.4]$ & 100.0 & 92.8\\
$[\geq 0.5]$ & 99.7 & 93.0\\
$[\geq 0.6]$ & 97.2 & 94.0\\
$[\geq 0.7]$ & 94.8 & 94.9\\
$[\geq 0.8]$ & 91.9 & 95.7\\
$[\geq 0.9]$ & 87.6 & 96.7\\
\hline 
\end{tabular}


\end{table}

\begin{table}[H]
\protect\caption{IMPUTE2's internal cross-validation for chromosome 18. Tables show the
percentage of concordance between genotyped calls and imputed calls
for 14,897,585 genotypes.\label{tab:cross_validation_chr_18}}

\centering

\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Nb Geno}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[0.0-0.1]$ & 0 & 0.0\\
$[0.1-0.2]$ & 0 & 0.0\\
$[0.2-0.3]$ & 0 & 0.0\\
$[0.3-0.4]$ & 0 & 0.0\\
$[0.4-0.5]$ & 40,611 & 44.6\\
$[0.5-0.6]$ & 324,021 & 52.6\\
$[0.6-0.7]$ & 324,001 & 60.2\\
$[0.7-0.8]$ & 383,487 & 68.5\\
$[0.8-0.9]$ & 577,849 & 77.2\\
$[0.9-1.0]$ & 13,247,616 & 97.1\\
\hline 
\end{tabular}
\hfill
\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Called (\%)}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[\geq 0.0]$ & 100.0 & 93.7\\
$[\geq 0.1]$ & 100.0 & 93.7\\
$[\geq 0.2]$ & 100.0 & 93.7\\
$[\geq 0.3]$ & 100.0 & 93.7\\
$[\geq 0.4]$ & 100.0 & 93.7\\
$[\geq 0.5]$ & 99.7 & 93.8\\
$[\geq 0.6]$ & 97.6 & 94.7\\
$[\geq 0.7]$ & 95.4 & 95.5\\
$[\geq 0.8]$ & 92.8 & 96.3\\
$[\geq 0.9]$ & 88.9 & 97.1\\
\hline 
\end{tabular}


\end{table}

\begin{table}[H]
\protect\caption{IMPUTE2's internal cross-validation for chromosome 19. Tables show the
percentage of concordance between genotyped calls and imputed calls
for 10,779,470 genotypes.\label{tab:cross_validation_chr_19}}

\centering

\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Nb Geno}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[0.0-0.1]$ & 0 & 0.0\\
$[0.1-0.2]$ & 0 & 0.0\\
$[0.2-0.3]$ & 0 & 0.0\\
$[0.3-0.4]$ & 0 & 0.0\\
$[0.4-0.5]$ & 41,248 & 44.9\\
$[0.5-0.6]$ & 295,389 & 52.2\\
$[0.6-0.7]$ & 290,234 & 59.6\\
$[0.7-0.8]$ & 340,564 & 67.1\\
$[0.8-0.9]$ & 499,530 & 75.8\\
$[0.9-1.0]$ & 9,312,505 & 96.3\\
\hline 
\end{tabular}
\hfill
\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Called (\%)}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[\geq 0.0]$ & 100.0 & 92.0\\
$[\geq 0.1]$ & 100.0 & 92.0\\
$[\geq 0.2]$ & 100.0 & 92.0\\
$[\geq 0.3]$ & 100.0 & 92.0\\
$[\geq 0.4]$ & 100.0 & 92.0\\
$[\geq 0.5]$ & 99.6 & 92.2\\
$[\geq 0.6]$ & 96.9 & 93.4\\
$[\geq 0.7]$ & 94.2 & 94.4\\
$[\geq 0.8]$ & 91.0 & 95.3\\
$[\geq 0.9]$ & 86.4 & 96.3\\
\hline 
\end{tabular}


\end{table}

\begin{table}[H]
\protect\caption{IMPUTE2's internal cross-validation for chromosome 20. Tables show the
percentage of concordance between genotyped calls and imputed calls
for 12,162,430 genotypes.\label{tab:cross_validation_chr_20}}

\centering

\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Nb Geno}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[0.0-0.1]$ & 0 & 0.0\\
$[0.1-0.2]$ & 0 & 0.0\\
$[0.2-0.3]$ & 0 & 0.0\\
$[0.3-0.4]$ & 0 & 0.0\\
$[0.4-0.5]$ & 36,326 & 44.5\\
$[0.5-0.6]$ & 283,357 & 52.6\\
$[0.6-0.7]$ & 279,857 & 60.2\\
$[0.7-0.8]$ & 331,909 & 68.1\\
$[0.8-0.9]$ & 503,444 & 76.9\\
$[0.9-1.0]$ & 10,727,537 & 96.9\\
\hline 
\end{tabular}
\hfill
\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Called (\%)}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[\geq 0.0]$ & 100.0 & 93.2\\
$[\geq 0.1]$ & 100.0 & 93.2\\
$[\geq 0.2]$ & 100.0 & 93.2\\
$[\geq 0.3]$ & 100.0 & 93.2\\
$[\geq 0.4]$ & 100.0 & 93.2\\
$[\geq 0.5]$ & 99.7 & 93.4\\
$[\geq 0.6]$ & 97.4 & 94.3\\
$[\geq 0.7]$ & 95.1 & 95.1\\
$[\geq 0.8]$ & 92.4 & 96.0\\
$[\geq 0.9]$ & 88.2 & 96.9\\
\hline 
\end{tabular}


\end{table}

\begin{table}[H]
\protect\caption{IMPUTE2's internal cross-validation for chromosome 21. Tables show the
percentage of concordance between genotyped calls and imputed calls
for 7,114,040 genotypes.\label{tab:cross_validation_chr_21}}

\centering

\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Nb Geno}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[0.0-0.1]$ & 0 & 0.0\\
$[0.1-0.2]$ & 0 & 0.0\\
$[0.2-0.3]$ & 0 & 0.0\\
$[0.3-0.4]$ & 0 & 0.0\\
$[0.4-0.5]$ & 21,851 & 44.4\\
$[0.5-0.6]$ & 158,870 & 52.5\\
$[0.6-0.7]$ & 156,774 & 60.2\\
$[0.7-0.8]$ & 186,361 & 68.5\\
$[0.8-0.9]$ & 282,650 & 77.3\\
$[0.9-1.0]$ & 6,307,534 & 97.0\\
\hline 
\end{tabular}
\hfill
\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Called (\%)}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[\geq 0.0]$ & 100.0 & 93.5\\
$[\geq 0.1]$ & 100.0 & 93.5\\
$[\geq 0.2]$ & 100.0 & 93.5\\
$[\geq 0.3]$ & 100.0 & 93.5\\
$[\geq 0.4]$ & 100.0 & 93.5\\
$[\geq 0.5]$ & 99.7 & 93.7\\
$[\geq 0.6]$ & 97.5 & 94.6\\
$[\geq 0.7]$ & 95.3 & 95.4\\
$[\geq 0.8]$ & 92.6 & 96.2\\
$[\geq 0.9]$ & 88.7 & 97.0\\
\hline 
\end{tabular}


\end{table}

\begin{table}[H]
\protect\caption{IMPUTE2's internal cross-validation for chromosome 22. Tables show the
percentage of concordance between genotyped calls and imputed calls
for 7,292,770 genotypes.\label{tab:cross_validation_chr_22}}

\centering

\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Nb Geno}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[0.0-0.1]$ & 0 & 0.0\\
$[0.1-0.2]$ & 0 & 0.0\\
$[0.2-0.3]$ & 0 & 0.0\\
$[0.3-0.4]$ & 0 & 0.0\\
$[0.4-0.5]$ & 25,940 & 44.4\\
$[0.5-0.6]$ & 184,146 & 51.9\\
$[0.6-0.7]$ & 180,014 & 59.3\\
$[0.7-0.8]$ & 211,940 & 67.2\\
$[0.8-0.9]$ & 314,770 & 76.1\\
$[0.9-1.0]$ & 6,375,960 & 96.6\\
\hline 
\end{tabular}
\hfill
\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Called (\%)}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[\geq 0.0]$ & 100.0 & 92.6\\
$[\geq 0.1]$ & 100.0 & 92.6\\
$[\geq 0.2]$ & 100.0 & 92.6\\
$[\geq 0.3]$ & 100.0 & 92.6\\
$[\geq 0.4]$ & 100.0 & 92.6\\
$[\geq 0.5]$ & 99.6 & 92.8\\
$[\geq 0.6]$ & 97.1 & 93.9\\
$[\geq 0.7]$ & 94.6 & 94.7\\
$[\geq 0.8]$ & 91.7 & 95.6\\
$[\geq 0.9]$ & 87.4 & 96.6\\
\hline 
\end{tabular}


\end{table}



\begin{table}[H]
\protect\caption{IMPUTE2's internal cross-validation across the genome. Tables show the
percentage of concordance between genotyped calls and imputed calls
for 492,578,415 genotypes.\label{tab:cross_validation}}

\centering

\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Nb Geno}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[0.0-0.1]$ & 0 & 0.0\\
$[0.1-0.2]$ & 0 & 0.0\\
$[0.2-0.3]$ & 0 & 0.0\\
$[0.3-0.4]$ & 0 & 0.0\\
$[0.4-0.5]$ & 1,091,601 & 44.5\\
$[0.5-0.6]$ & 9,259,754 & 53.0\\
$[0.6-0.7]$ & 9,281,067 & 60.9\\
$[0.7-0.8]$ & 11,101,124 & 69.2\\
$[0.8-0.9]$ & 16,903,475 & 78.1\\
$[0.9-1.0]$ & 444,941,394 & 97.5\\
\hline 
\end{tabular}
\hfill
\begin{tabular}{crr}
\hline 
\multicolumn{1}{c}{\textbf{Interval}}
 & \multicolumn{1}{c}{\textbf{Called (\%)}}
 & \multicolumn{1}{c}{\textbf{Concordance (\%)}}
\\
\hline 

$[\geq 0.0]$ & 100.0 & 94.5\\
$[\geq 0.1]$ & 100.0 & 94.5\\
$[\geq 0.2]$ & 100.0 & 94.5\\
$[\geq 0.3]$ & 100.0 & 94.5\\
$[\geq 0.4]$ & 100.0 & 94.5\\
$[\geq 0.5]$ & 99.8 & 94.6\\
$[\geq 0.6]$ & 97.9 & 95.4\\
$[\geq 0.7]$ & 96.0 & 96.1\\
$[\geq 0.8]$ & 93.8 & 96.8\\
$[\geq 0.9]$ & 90.3 & 97.5\\
\hline 
\end{tabular}


\end{table}


\pagebreak


\subsection{Completion rate}\label{subsec:completion_rate}


To evaluate the completion rate, we first used a probability threshold of
$\geq 90.0$\%, which means that a genotype must have one of
the three allele combination (\texttt{AA}, \texttt{AB} or \texttt{BB})
probabilities higher or equal to 90.0\% to be considered as a
\textit{good call}.\\

For the 13,772,014 imputed variants, an average completion rate of
95.9\% was obtained. When removing variants with an
information value under 0.00, and a completion rate under
98.0\%, 9,423,875 (68.4\%)
markers were left, with an average completion rate of
99.6\%, meaning that there is a mean of
5.6 missing genotypes (for 1,465 samples) for
each markers.\\

A total of 364,650 variants were previously genotyped,
125,013 (34.3\%) of
which had  a call rate lower than 100\% (\textit{i.e.} 298,133
missing genotypes). A total of 298,133
(100.0\%) missing genotypes were imputed with high
quality (\textit{i.e.} 364,650 markers now have a call
rate of 100\%).


\subsection{Minor allele frequencies}\label{subsec:maf}


Out of the 9,423,875 imputed variants with a completion rate
$\geq 98.0\%$, there were 3,692,736
(39.2\%) variants with a minor
allele frequency (MAF) $\geq 1\%$, 2,781,392
(29.5\%) variants with a MAF $\geq 5\%$, and
6,642,483 (70.5\%) variants with a MAF
$<5\%$.\\


\section{Conclusions}\label{sec:conclusions}


Statistical analyses will be performed with the genome-wide imputed dataset,
which include 9,423,875 imputed variants (done with an information
threshold of $\geq0.00$, and a completion rate of
$\geq98.0$\% at an imputation probability threshold of
$\geq90.0$\%). This total includes 364,650
previously genotyped variants.\\

All files were generated in the \texttt{genipe} directory and were
separated by chromosomes (\texttt{genipe/chr*} directories). The
final (merged) results (generated by IMPUTE2) are located in the
\texttt{genipe/chr*/final\_impute2} directories. All the output files are
described below.


\begin{itemize}

\item \texttt{chr*.imputed.alleles}: description of the reference and
alternative allele at each site.


\item \texttt{chr*.imputed.completion\_rates}: number of missing values and
completion rate for all site (using a probability threshold $\geq
90.0\%$).


\item \texttt{chr*.imputed.good\_sites}: list of sites which pass the
information threshold ($\geq 0.00$) and the completion rate threshold
($\geq 98.0\%$) using the probability threshold $\geq 90.0\%$.


\item \texttt{chr*.imputed.impute2}: imputation results (merged from all
segments).


\item \texttt{chr*.imputed.impute2\_info}: the IMPUTE2 marker-wise
information file (merged from all segments).


\item \texttt{chr*.imputed.imputed\_sites}: list of imputed sites (excluding
sites that were previously genotyped in the study cohort).


\item \texttt{chr*.imputed.log}: log file of the merging procedure.


\item \texttt{chr*.imputed.maf}: minor allele frequency (along with minor
allele identification) for all sites using the probability threshold
$\geq 90.0\%$.


\item \texttt{chr*.imputed.map}: a map file describing the genomic location
of all sites.


\item \texttt{chr*.imputed.sample}: the sample file generated by the phasing
step.

\end{itemize}





\bibliographystyle{references}
\phantomsection\addcontentsline{toc}{section}{\refname}%
\bibliography{references}

\clearpage

\phantomsection\addcontentsline{toc}{section}{Annex I: Execution time}%
\section*{Annex I: Execution times}\label{sec:execution_time}

The following tables show the execution time required by all the
different tasks. All tasks are split by chromosomes. Execution times
for imputation for each chromosome are means of individual segment
times. Computing all genotyped markers' missing rate took no time.


\begin{table}[H]
\protect\caption{Execution time for the '\texttt{plink\_exclude\_chr*}' tasks.\label{tab:plink_exclude_exec_time}}

\centering

\begin{tabular}{rr}
\hline 
\multicolumn{1}{c}{\textbf{Chrom}}
 & \multicolumn{1}{c}{\textbf{Time}}
\\
\hline 

1 & {\color{light_gray}00:00:0}3\\
2 & {\color{light_gray}00:00:0}3\\
3 & {\color{light_gray}00:00:0}1\\
4 & {\color{light_gray}00:00:0}1\\
5 & {\color{light_gray}00:00:0}2\\
6 & {\color{light_gray}00:00:0}4\\
7 & {\color{light_gray}00:00:0}3\\
8 & {\color{light_gray}00:00:0}2\\
9 & {\color{light_gray}00:00:0}3\\
10 & {\color{light_gray}00:00:0}3\\
11 & {\color{light_gray}00:00:0}3\\
\hline 
\end{tabular}
\hspace{1cm}
\begin{tabular}{rr}
\hline 
\multicolumn{1}{c}{\textbf{Chrom}}
 & \multicolumn{1}{c}{\textbf{Time}}
\\
\hline 

12 & {\color{light_gray}00:00:0}3\\
13 & {\color{light_gray}00:00:0}2\\
14 & {\color{light_gray}00:00:0}3\\
15 & {\color{light_gray}00:00:0}2\\
16 & {\color{light_gray}00:00:0}2\\
17 & {\color{light_gray}00:00:0}2\\
18 & {\color{light_gray}00:00:0}2\\
19 & {\color{light_gray}00:00:0}1\\
20 & {\color{light_gray}00:00:0}1\\
21 & {\color{light_gray}00:00:0}1\\
22 & {\color{light_gray}00:00:0}1\\
\hline 
\end{tabular}


\end{table}

\begin{table}[H]
\protect\caption{Execution time for the '\texttt{shapeit\_check\_chr*\_1}' tasks.\label{tab:shapeit_check_1_exec_time}}

\centering

\begin{tabular}{rr}
\hline 
\multicolumn{1}{c}{\textbf{Chrom}}
 & \multicolumn{1}{c}{\textbf{Time}}
\\
\hline 

1 & {\color{light_gray}00:00:}21\\
2 & {\color{light_gray}00:00:}23\\
3 & {\color{light_gray}00:00:}20\\
4 & {\color{light_gray}00:00:}20\\
5 & {\color{light_gray}00:00:}18\\
6 & {\color{light_gray}00:00:}17\\
7 & {\color{light_gray}00:00:}15\\
8 & {\color{light_gray}00:00:}15\\
9 & {\color{light_gray}00:00:}12\\
10 & {\color{light_gray}00:00:}13\\
11 & {\color{light_gray}00:00:}13\\
\hline 
\end{tabular}
\hspace{1cm}
\begin{tabular}{rr}
\hline 
\multicolumn{1}{c}{\textbf{Chrom}}
 & \multicolumn{1}{c}{\textbf{Time}}
\\
\hline 

12 & {\color{light_gray}00:00:}13\\
13 & {\color{light_gray}00:00:0}9\\
14 & {\color{light_gray}00:00:0}9\\
15 & {\color{light_gray}00:00:0}8\\
16 & {\color{light_gray}00:00:0}9\\
17 & {\color{light_gray}00:00:0}6\\
18 & {\color{light_gray}00:00:0}6\\
19 & {\color{light_gray}00:00:0}6\\
20 & {\color{light_gray}00:00:0}5\\
21 & {\color{light_gray}00:00:0}4\\
22 & {\color{light_gray}00:00:0}4\\
\hline 
\end{tabular}


\end{table}

\begin{table}[H]
\protect\caption{Execution time for the '\texttt{plink\_flip\_chr*}' tasks.\label{tab:plink_flip_exec_time}}

\centering

\begin{tabular}{rr}
\hline 
\multicolumn{1}{c}{\textbf{Chrom}}
 & \multicolumn{1}{c}{\textbf{Time}}
\\
\hline 

1 & {\color{light_gray}00:00:0}4\\
2 & {\color{light_gray}00:00:0}5\\
3 & {\color{light_gray}00:00:0}4\\
4 & {\color{light_gray}00:00:0}4\\
5 & {\color{light_gray}00:00:0}3\\
6 & {\color{light_gray}00:00:0}5\\
7 & {\color{light_gray}00:00:0}2\\
8 & {\color{light_gray}00:00:0}3\\
9 & {\color{light_gray}00:00:0}3\\
10 & {\color{light_gray}00:00:0}3\\
11 & {\color{light_gray}00:00:0}3\\
\hline 
\end{tabular}
\hspace{1cm}
\begin{tabular}{rr}
\hline 
\multicolumn{1}{c}{\textbf{Chrom}}
 & \multicolumn{1}{c}{\textbf{Time}}
\\
\hline 

12 & {\color{light_gray}00:00:0}3\\
13 & {\color{light_gray}00:00:0}2\\
14 & {\color{light_gray}00:00:0}2\\
15 & {\color{light_gray}00:00:0}2\\
16 & {\color{light_gray}00:00:0}2\\
17 & {\color{light_gray}00:00:0}2\\
18 & {\color{light_gray}00:00:0}2\\
19 & {\color{light_gray}00:00:0}1\\
20 & {\color{light_gray}00:00:0}1\\
21 & {\color{light_gray}00:00:0}1\\
22 & {\color{light_gray}00:00:0}1\\
\hline 
\end{tabular}


\end{table}

\begin{table}[H]
\protect\caption{Execution time for the '\texttt{shapeit\_check\_chr*\_2}' tasks.\label{tab:shapeit_check_2_exec_time}}

\centering

\begin{tabular}{rr}
\hline 
\multicolumn{1}{c}{\textbf{Chrom}}
 & \multicolumn{1}{c}{\textbf{Time}}
\\
\hline 

1 & {\color{light_gray}00:00:}22\\
2 & {\color{light_gray}00:00:}23\\
3 & {\color{light_gray}00:00:}18\\
4 & {\color{light_gray}00:00:}19\\
5 & {\color{light_gray}00:00:}17\\
6 & {\color{light_gray}00:00:}17\\
7 & {\color{light_gray}00:00:}15\\
8 & {\color{light_gray}00:00:}15\\
9 & {\color{light_gray}00:00:}12\\
10 & {\color{light_gray}00:00:}13\\
11 & {\color{light_gray}00:00:}13\\
\hline 
\end{tabular}
\hspace{1cm}
\begin{tabular}{rr}
\hline 
\multicolumn{1}{c}{\textbf{Chrom}}
 & \multicolumn{1}{c}{\textbf{Time}}
\\
\hline 

12 & {\color{light_gray}00:00:}12\\
13 & {\color{light_gray}00:00:0}9\\
14 & {\color{light_gray}00:00:0}9\\
15 & {\color{light_gray}00:00:0}8\\
16 & {\color{light_gray}00:0}1:22\\
17 & {\color{light_gray}00:00:0}6\\
18 & {\color{light_gray}00:00:0}7\\
19 & {\color{light_gray}00:00:0}6\\
20 & {\color{light_gray}00:00:0}5\\
21 & {\color{light_gray}00:00:0}4\\
22 & {\color{light_gray}00:00:0}4\\
\hline 
\end{tabular}


\end{table}

\begin{table}[H]
\protect\caption{Execution time for the '\texttt{plink\_final\_exclude\_chr*}' tasks.\label{tab:plink_final_exclude_exec_time}}

\centering

\begin{tabular}{rr}
\hline 
\multicolumn{1}{c}{\textbf{Chrom}}
 & \multicolumn{1}{c}{\textbf{Time}}
\\
\hline 

1 & {\color{light_gray}00:00:0}4\\
2 & {\color{light_gray}00:00:0}5\\
3 & {\color{light_gray}00:00:0}4\\
4 & {\color{light_gray}00:00:0}4\\
5 & {\color{light_gray}00:00:0}3\\
6 & {\color{light_gray}00:00:0}5\\
7 & {\color{light_gray}00:00:0}3\\
8 & {\color{light_gray}00:00:0}2\\
9 & {\color{light_gray}00:00:0}3\\
10 & {\color{light_gray}00:00:0}3\\
11 & {\color{light_gray}00:00:0}3\\
\hline 
\end{tabular}
\hspace{1cm}
\begin{tabular}{rr}
\hline 
\multicolumn{1}{c}{\textbf{Chrom}}
 & \multicolumn{1}{c}{\textbf{Time}}
\\
\hline 

12 & {\color{light_gray}00:00:0}3\\
13 & {\color{light_gray}00:00:0}2\\
14 & {\color{light_gray}00:00:0}2\\
15 & {\color{light_gray}00:00:0}2\\
16 & {\color{light_gray}00:00:0}2\\
17 & {\color{light_gray}00:00:0}2\\
18 & {\color{light_gray}00:00:0}1\\
19 & {\color{light_gray}00:00:0}1\\
20 & {\color{light_gray}00:00:0}1\\
21 & {\color{light_gray}00:00:0}1\\
22 & {\color{light_gray}00:00:0}1\\
\hline 
\end{tabular}


\end{table}

\begin{table}[H]
\protect\caption{Execution time for the '\texttt{shapeit\_phase\_chr*}' tasks.\label{tab:shapeit_phase_exec_time}}

\centering

\begin{tabular}{rr}
\hline 
\multicolumn{1}{c}{\textbf{Chrom}}
 & \multicolumn{1}{c}{\textbf{Time}}
\\
\hline 

1 & {\color{light_gray}0}6:29:43\\
2 & {\color{light_gray}0}6:41:10\\
3 & {\color{light_gray}0}5:36:32\\
4 & {\color{light_gray}0}5:09:59\\
5 & {\color{light_gray}0}4:45:12\\
6 & {\color{light_gray}0}5:42:38\\
7 & {\color{light_gray}0}4:29:33\\
8 & {\color{light_gray}0}4:07:35\\
9 & {\color{light_gray}0}3:30:52\\
10 & {\color{light_gray}0}3:58:40\\
11 & {\color{light_gray}0}3:48:29\\
\hline 
\end{tabular}
\hspace{1cm}
\begin{tabular}{rr}
\hline 
\multicolumn{1}{c}{\textbf{Chrom}}
 & \multicolumn{1}{c}{\textbf{Time}}
\\
\hline 

12 & {\color{light_gray}0}3:49:25\\
13 & {\color{light_gray}0}2:50:17\\
14 & {\color{light_gray}0}2:33:05\\
15 & {\color{light_gray}0}2:25:40\\
16 & {\color{light_gray}0}2:32:13\\
17 & {\color{light_gray}0}2:10:11\\
18 & {\color{light_gray}0}1:49:07\\
19 & {\color{light_gray}0}1:23:41\\
20 & {\color{light_gray}0}1:36:27\\
21 & {\color{light_gray}0}1:00:05\\
22 & {\color{light_gray}00:}52:24\\
\hline 
\end{tabular}


\end{table}

\begin{table}[H]
\protect\caption{Execution time for the '\texttt{impute2\_chr*}' tasks.\label{tab:impute2_exec_time}}

\centering

\begin{tabular}{rrrr}
\hline 
\multicolumn{1}{c}{\textbf{Chrom}}
 & \multicolumn{1}{c}{\textbf{Nb Seg.}}
 & \multicolumn{1}{c}{\textbf{Mean T.}}
 & \multicolumn{1}{c}{\textbf{Max T.}}
\\
\hline 

1 & 50 & {\color{light_gray}00:}19:14 & {\color{light_gray}00:}54:23\\
2 & 49 & {\color{light_gray}00:}16:07 & {\color{light_gray}00:}33:22\\
3 & 40 & {\color{light_gray}00:}19:40 & {\color{light_gray}00:}59:25\\
4 & 39 & {\color{light_gray}00:}18:01 & {\color{light_gray}00:}34:49\\
5 & 37 & {\color{light_gray}00:}17:37 & {\color{light_gray}00:}31:28\\
6 & 35 & {\color{light_gray}00:}16:39 & {\color{light_gray}00:}29:11\\
7 & 32 & {\color{light_gray}00:}17:12 & {\color{light_gray}00:}35:35\\
8 & 30 & {\color{light_gray}00:}16:54 & {\color{light_gray}00:}27:23\\
9 & 29 & {\color{light_gray}00:}13:07 & {\color{light_gray}00:}26:00\\
10 & 28 & {\color{light_gray}00:}15:14 & {\color{light_gray}00:}26:09\\
11 & 27 & {\color{light_gray}00:}13:03 & {\color{light_gray}00:}24:06\\
\hline 
\end{tabular}
\hspace{1cm}
\begin{tabular}{rrrr}
\hline 
\multicolumn{1}{c}{\textbf{Chrom}}
 & \multicolumn{1}{c}{\textbf{Nb Seg.}}
 & \multicolumn{1}{c}{\textbf{Mean T.}}
 & \multicolumn{1}{c}{\textbf{Max T.}}
\\
\hline 

12 & 27 & {\color{light_gray}00:}13:06 & {\color{light_gray}00:}22:43\\
13 & 24 & {\color{light_gray}00:0}9:40 & {\color{light_gray}00:}16:37\\
14 & 22 & {\color{light_gray}00:0}9:04 & {\color{light_gray}00:}13:00\\
15 & 21 & {\color{light_gray}00:0}8:30 & {\color{light_gray}00:}13:52\\
16 & 19 & {\color{light_gray}00:}10:01 & {\color{light_gray}00:}19:01\\
17 & 17 & {\color{light_gray}00:}10:01 & {\color{light_gray}00:}13:55\\
18 & 16 & {\color{light_gray}00:}10:54 & {\color{light_gray}00:}14:00\\
19 & 12 & {\color{light_gray}00:}10:47 & {\color{light_gray}00:}13:27\\
20 & 13 & {\color{light_gray}00:}10:14 & {\color{light_gray}00:}12:52\\
21 & 10 & {\color{light_gray}00:0}7:20 & {\color{light_gray}00:}13:33\\
22 & 11 & {\color{light_gray}00:0}7:35 & {\color{light_gray}00:}15:59\\
\hline 
\end{tabular}


\end{table}

\begin{table}[H]
\protect\caption{Execution time for the '\texttt{merge\_impute2\_chr*}' tasks.\label{tab:merge_impute2_exec_time}}

\centering

\begin{tabular}{rr}
\hline 
\multicolumn{1}{c}{\textbf{Chrom}}
 & \multicolumn{1}{c}{\textbf{Time}}
\\
\hline 

1 & {\color{light_gray}00:}34:44\\
2 & {\color{light_gray}00:}36:47\\
3 & {\color{light_gray}00:}31:14\\
4 & {\color{light_gray}00:}31:42\\
5 & {\color{light_gray}00:}27:45\\
6 & {\color{light_gray}00:}28:42\\
7 & {\color{light_gray}00:}26:04\\
8 & {\color{light_gray}00:}24:01\\
9 & {\color{light_gray}00:}19:14\\
10 & {\color{light_gray}00:}22:14\\
11 & {\color{light_gray}00:}21:35\\
\hline 
\end{tabular}
\hspace{1cm}
\begin{tabular}{rr}
\hline 
\multicolumn{1}{c}{\textbf{Chrom}}
 & \multicolumn{1}{c}{\textbf{Time}}
\\
\hline 

12 & {\color{light_gray}00:}21:09\\
13 & {\color{light_gray}00:}16:02\\
14 & {\color{light_gray}00:}14:31\\
15 & {\color{light_gray}00:}13:01\\
16 & {\color{light_gray}00:}13:46\\
17 & {\color{light_gray}00:}11:33\\
18 & {\color{light_gray}00:0}9:54\\
19 & {\color{light_gray}00:0}9:50\\
20 & {\color{light_gray}00:0}7:53\\
21 & {\color{light_gray}00:0}5:29\\
22 & {\color{light_gray}00:0}5:19\\
\hline 
\end{tabular}


\end{table}


\end{document}
